README.md

```diff
@@ -1,73 +1,86 @@
-pmdk: Persistent Memory Development Kit
-=======================================
+# **PMDK: Persistent Memory Development Kit**
 
 [![Build Status](https://travis-ci.org/pmem/pmdk.svg?branch=master)](https://travis-ci.org/pmem/pmdk)
 [![Build status](https://ci.appveyor.com/api/projects/status/u2l1db7ucl5ktq10/branch/master?svg=true&pr=false)](https://ci.appveyor.com/project/pmem/pmdk/branch/master)
 [![Coverity Scan Build Status](https://img.shields.io/coverity/scan/3015.svg)](https://scan.coverity.com/projects/pmem-pmdk)
 [![PMDK release version](https://img.shields.io/github/release/pmem/pmdk.svg)](https://github.com/pmem/pmdk/releases/latest)
 [![Coverage Status](https://codecov.io/github/pmem/pmdk/coverage.svg?branch=master)](https://codecov.io/gh/pmem/pmdk/branch/master)
+[Join the discussion in Google Groups](http://groups.google.com/group/pmem)
```

Comment:
this line is a row of "shields"  - google group does not fit here

Reply:
I just removed the google groups link from that row.